cmd/openshift-install/gather.go

```diff
@@ -151,19 +172,27 @@ func gatherBootstrap(bootstrap string, port int, masters []string, directory str
 		return "", errors.Wrap(err, "failed to create SSH client")
 	}
 
-	gatherID := time.Now().Format("20060102150405")
 	if err := ssh.Run(client, fmt.Sprintf("/usr/local/bin/installer-gather.sh --id %s %s", gatherID, strings.Join(masters, " "))); err != nil {
 		return "", errors.Wrap(err, "failed to run remote command")
 	}
-	file := filepath.Join(directory, fmt.Sprintf("log-bundle-%s.tar.gz", gatherID))
+
+	file := filepath.Join(directory, fmt.Sprintf("cluster-log-bundle-%s.tar.gz", gatherID))
 	if err := ssh.PullFileTo(client, fmt.Sprintf("/home/core/log-bundle-%s.tar.gz", gatherID), file); err != nil {
 		return "", errors.Wrap(err, "failed to pull log file from remote")
 	}
-	path, err := filepath.Abs(file)
+
+	clusterLogBundlePath, err := filepath.Abs(file)
 	if err != nil {
 		return "", errors.Wrap(err, "failed to stat log file")
 	}
-	return path, nil
+
+	logBundlePath := filepath.Join(filepath.Dir(clusterLogBundlePath), fmt.Sprintf("log-bundle-%s.tar.gz", gatherID))
+	err = gather.CombineArchives(logBundlePath, map[string]string{serialLogBundlePath: "serial"})
```

Comment:
I'm curious to see what is happening here. The CombineArchive() function just adds a subdirectory named "serial" and places the serial logs under there. The directory structure remains the same.

Reply:
Are you getting different results in your testing?

Reply:
@jhixson74 it occurred to me that we should actually be able to test this in CI. If you add a DO NOT MERGE commit which would cause bootstrap to fail, say: break a critical manifest, or cause an error in bootkube.sh, then CI should run gather bootstrap and collect the bundle in CI, so reviewers can examine it there.